### PR TITLE
[FIX] depend on inexisting module make openupgrade failing if module is installed in v 8.0

### PR DESCRIPTION
**Pull request by legalsylvain**
the commit remove the dependency on a unexisting module. 
Note that the web_graph_improved module is **NOT** installable on the version 9.0

summary of the problem : 

- use a V8 database
- installed web_graph_improved 
- migrate on v9, making an update all 

odoo will scan all the module in v9 version to create dependencies tree, **including uninstallable modules**, so will fail because the module web_graph_improved depends on web_graph that doesn't exist in V9.

CC : @OCA/openupgrade-maintainers

### CODE REVIEW

File: web_graph_improved/__openerp__.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
     'author': "Leonardo Donelli,Odoo Community Association (OCA)",
     'license': 'AGPL-3',
     'website': 'http://learts.glaucus.in',
-    'depends': ['web_graph'],
+    'depends': ['web'],
     'data': ['view/graph_improved_view.xml'],
     'installable': False,
```
Comment (MiquelRForgeFlow) [inline web_graph_improved/__openerp__.py]: 🤔

Comment (legalsylvain) [reply web_graph_improved/__openerp__.py]: Hi @mreficent, your review is quite cryptic !
could you be more explicit ? 
regards.

Comment (MiquelRForgeFlow) [reply web_graph_improved/__openerp__.py]: It's a bit odd doing changes in a module that is not installable 😏

Comment (legalsylvain) [reply web_graph_improved/__openerp__.py]: yes ! but even if it's not installable, it is read by odoo, and as it depends on a non existing module, it make the upgrade failing. 
The alternative was to delete fully this module, that has nothing to do in V9 serie. I thought it was less brutal, just to remove the undesired dependency.

Comment (MiquelRForgeFlow) [reply web_graph_improved/__openerp__.py]: I think the best solution is just delete this deprecated module from the repo! :)

Comment (MiquelRForgeFlow) [reply web_graph_improved/__openerp__.py]: Please, delete it. And also, you could delete it from v10 too :)

Comment (legalsylvain) [reply web_graph_improved/__openerp__.py]: > Please, delete it.

OK done.

> And also, you could delete it from v10 too :)

I'll do it when I'll work on 9.0 -> 10.0 migration.

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**